cfg/kde.cfg

```diff
@@ -48,4 +48,19 @@
       <not-uninit/>
     </arg>
   </function>
+  <!-- QString i18n(const char*) -->
+  <!-- QString ki18n(const char*) -->
+  <function name="i18n,ki18n">
+    <noreturn>false</noreturn>
```

Comment:
This should probably have `<leak-ignore/>` and `<pure/>` as well.

Reply:
Was expecting that I forgot something. Will look into it.

Reply:
Done.